app/code/Magento/InventoryInStorePickupAdminUi/Plugin/Sales/Model/AdminOrder/Create/SetPickupLocationFromPost.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupAdminUi\Plugin\Sales\Model\AdminOrder\Create;
```

Comment:
Please change namespace with mention of corresponding class which it intercepts.
This class has all indications to be a part of Presentation Layer.

Reply:
Ok